src/utils/generate_pot.sh

```diff
@@ -49,7 +49,10 @@ cd "${repo_dir}"
 	>> "${temp_pot_file}"
 
 find 'src/cgame' 'src/sgame' 'src/shared' \
-	-name '*.cpp' -a ! -name sg_admin.cpp -a ! -name sg_cmds.cpp \
+	-name '*.cpp' \
+	-a ! -name sg_admin.cpp \
+	-a ! -name sg_cmds.cpp \
+	-a ! -name sg_maprotation.cpp \
```

Comment:
You have to add this to the xgettext command at the end

Reply:
Good catch! Fixed.